packages/calypso-build/webpack.config.js

```diff
@@ -121,6 +121,9 @@ function getWebpackConfig(
 		plugins: [
 			new webpack.DefinePlugin( {
 				'process.env.NODE_ENV': JSON.stringify( process.env.NODE_ENV ),
+				'process.env.FORCE_REDUCED_MOTION': JSON.stringify(
+					!! process.env.FORCE_REDUCED_MOTION || false
```

Comment:
`@wordpress/compose` requires this (or `node: true`) in webpack config.

https://github.com/WordPress/gutenberg/pull/16679